SendKeys_Accept.vbs

```diff
@@ -1,18 +1,18 @@
-Set WSHShell = WScript.CreateObject("WScript.Shell")
-WScript.Sleep(5000)
-WScript.Sleep(5000)
-
-WScript.Sleep(5000)
-WScript.Sleep(5000)
```

Comment:
There are “diffs” in unchanged sections and files.
(probably due to line break code changes)